src/test/resources/output/json/00216/TT002165019.json

```diff
@@ -0,0 +1,57 @@
+{
+  "@context" : "http://lobid.org/resources/context.jsonld",
+  "contribution" : [ {
+    "agent" : {
+      "id" : "http://d-nb.info/gnd/104698489",
+      "label" : "Boskamp, Aloys Erich",
+      "type" : [ "Person" ]
+    },
+    "role" : {
+      "id" : "http://id.loc.gov/vocabulary/relators/ctb",
+      "label" : "Mitwirkende"
+    },
+    "type" : [ "Contribution" ]
+  } ],
+  "contributorOrder" : [ "http://d-nb.info/gnd/104698489" ],
+  "describedBy" : {
+    "dateCreated" : "20060718",
+    "id" : "http://lobid.org/resources/TT002165019",
+    "label" : "lobid Ressource"
+  },
+  "extent" : "S. 77",
+  "hasItem" : [ {
+    "callNumber" : "G m 38",
+    "describedBy" : {
+      "id" : "http://lobid.org/items/TT002165019:DE-6-139a:G%20m%2038",
+      "label" : "lobid Bestandsressource"
+    },
+    "heldBy" : {
+      "id" : "http://lobid.org/organisations/DE-6-139a",
+      "label" : "lobid Organisation"
+    },
+    "id" : "http://lobid.org/items/TT002165019:DE-6-139a:G%20m%2038#!",
+    "itemOf" : "http://lobid.org/resources/TT002165019#!",
+    "label" : "lobid Bestandsressource",
+    "note" : "00193001",
+    "type" : [ "Item" ]
+  } ],
+  "hbzId" : "TT002165019",
+  "id" : "http://lobid.org/resources/TT002165019#!",
+  "medium" : [ {
+    "id" : "http://rdaregistry.info/termList/RDAproductionMethod/1010",
+    "label" : "Print"
+  } ],
+  "otherTitleInformation" : [ "Jüdisches Schicksal in Paderborn" ],
+  "publication" : [ {
+    "location" : "Paderborn",
+    "startDate" : "1964",
+    "type" : [ "PublicationEvent" ]
+  } ],
+  "responsibilityStatement" : [ "Hrsg.: Stadt Paderborn. Verantwortl.: A. Erich Boskamp" ],
+  "sameAs" : [ {
+    "id" : "http://hub.culturegraph.org/resource/HBZ-TT002165019",
+    "label" : "Culturegraph Ressource"
+  } ],
+  "title" : "Bauen wir doch aufs neue das alte Haus",
+  "type" : [ "BibliographicResource", "Book" ]
+}
```

Comment:
Ah -forget my comment about the item infos. They reside here as for all the other resources because this JSON-LD file does not reflect the ES index.